app/models/foreman_tasks/task_group.rb

```diff
@@ -0,0 +1,8 @@
+module ForemanTasks
+  class TaskGroup < ActiveRecord::Base
+
+    has_and_belongs_to_many :tasks,
```

Comment:
I have pretty bad experience with habtm, could we introduce `TaskGroupMember` model, or something like that and use `has_many :though` instead?